readme.txt

```diff
@@ -1,12 +1,53 @@
-=== inseri core ===
-Contributors: inseri swiss
-Tags: block, inseri
-Requires at least: 5.8
-Requires PHP: 7.0
+=== Scientific and Interactive Blocks - inseri core ===
+Contributors: inseriswiss
+Tags: Gutenberg blocks, WordPress blocks, gutenberg, blocks, inseri
+Requires at least: 5.6
+Tested up to: 6.2
+Requires PHP: 5.6
+Stable tag: 0.1.0
 License: GPL-3.0-or-later
 License URI: https://www.gnu.org/licenses/gpl-3.0.html
 
-short desc
+Scientific and Interactive Gutenberg Blocks to facilitate Open Science
 
 == Description ==
-long desc
+
+The main focus of inseri.swiss is to provide tools for scientific publications and to promote Open Access, in other words, to eliminate technical barriers and provide reproducibility. As a result, the inseri core provides special blocks that enable to share data between blocks.
```

Comment:
I would start with the vision - https://inseri.swiss/about/#vision :
> The vision of inseri is to connect open data providers, interactive online components, web APIs, and local user data so that a user can combine them in any possible way. It serves as an online collaborative and publishing platform that allows the researchers to publish open, FAIR, citable, interactive, and executable web content with ease for effortless re-use by any researcher or the general public.

Add the short description and some generic words about inseri core - I reused the some text from the introduction - https://docs.inseri.swiss/ :
> inseri core introduces scientific and interactive Gutenberg blocks to facilitate open science. The inseri core blocks are not isolated elements but they can receive input from and give output to other blocks. A typical example would start with a text block containing the configuration for a dropdown. Selecting one option in the dropdown would provide an API block with the parameters to query data from an external source. Using a Python block, the response could then be transformed into a format which a block for graphs can display.

Reply:
I think the vision is little bit outdated: for example, inseri is not a platform anymore
and chaining of too much adjective can also be overwhelming for the reader
that is why I tried to rephrase the vision part:
```

The vision of inseri is to enable scholars to design web pages without any prior knowledge. So that they can publish their research findings in an open, citable and interactive way in combination with own their data or any public data repository. Through interactivity, everyone is encouraged to play around with the parameters and use their own data to better reproduce the research findings.

As a result, inseri core introduces scientific and interactive Gutenberg blocks to facilitate open access. The inseri core blocks are not isolated elements but they can receive input from and give output to other blocks. A typical example would start with a Text Editor block containing the configuration for a Dropdown block. By selecting one option in the Dropdown block would provide a Web API block with the parameters to retrieve data from an external source. Using a Python Code block, the web API response could then be transformed into a suitable data format for a Plotly Chart block.
```

Reply:
I see your point. So far I will avoid mentioning the reproducibility so the last sentence from the first paragraph:

> Through interactivity, everyone is encouraged to play around with the parameters and use their own data to ~~better reproduce~~ **explore** the research findings."